integration/okteto.go

```diff
@@ -87,7 +87,7 @@ func GetTestNamespace(prefix, user string) string {
 	} else {
 		os = os[:3]
 	}
-	namespace := fmt.Sprintf("%s-%s-%d-%s", prefix, os, time.Now().Unix(), user)
+	namespace := fmt.Sprintf("%s-%s-%s-%d", user, prefix, os, time.Now().Unix())
```

Comment:
@jLopezbarb @andreafalzetti i propose to change the test namespace string so it starts with the user, as i move accross environments and in my devcluster the namespace has to start with my user.... do you think this can break ? let me know your thoughts! thanks

Reply:
Can we create a function and based on an environment  change the behaviour? On staging the user has to be at the end and given that we passed the tests on staging everything will fail

Reply:
i removed the commit, we can think about this when we improve integration. thanks